applications/tari_console_wallet/src/ui/state/app_state.rs

```diff
@@ -248,10 +248,7 @@ impl AppState {
     // Return alias or pub key if the contact is not in the list.
     pub fn get_alias(&self, pub_key: &RistrettoPublicKey) -> String {
         let pub_key_hex = format!("{}", pub_key);
-        // TODO: We can uncomment this to indicated unknown origin, otherwise there is our pub key.
```

Comment:
Don't think the Unknown case is useful really. If you make a contact with your own pubkey that is fine you will just send transactions to yourself which is a feature that works.